pkg/naming/constants.go

```diff
@@ -132,3 +142,17 @@ const (
 const (
 	ScyllaRuntimeConfigKey string = "ScyllaRuntimeConfig"
 )
+
+const (
+	PortNameCQL              = "cql"
+	PortNameCQLSSL           = "cql-ssl"
+	PortNameCQLShardAware    = "cql-shard-aware"
+	PortNameCQLSSLShardAware = "cql-ssl-shard-aware"
+	PortNameAlternator       = "alternator"
+	PortNameThrift           = "thrift"
+)
+
+const (
+	BackendCQL        = "cql"
```

Comment:
these are also limited to a single function - is there a reason to export them?

Reply:
done, moved to unexported consts in resource.go